{{Introduction to Python Basics: Variables and Data types}}/.md

```diff
@@ -0,0 +1,180 @@
+
+Ex 1.6:
+
+	string = '42'
+
+This is another example of a string. Although we would think that ‘42’ is an integer that is not the case for this particular example. Since the number 42 is in quotation marks it is actually considered a string. Any numbers that are enclosed in quotation marks are considered string. 
+
+Ex 1.7:
+
+	string = '42.0'
+
+This example is similar to the example above except the value 42.0 is considered a floating number because of the decimal. However, the same logic is applied to this example. Since the value is enclosed in single quotation marks the floating number 42.0 is actually the string ‘42.0’.
+
+Test your knowledge:
+
+	32.189 is considered what kind of Data Type?
+	
+	‘90’ is a number. True or False?
+	
+	“Hello, what is your name?’ is an acceptable string. True or False?
+	
+	-78 is what kind of Data Type?
+
+
+
+
+II. Variables
+
+Now that we have learned about Data Types we will dive into variables. I want you to think of Python variables like a box. The box itself is the variable and you can place a single value into the box. The values stored inside can be any of the Data Types that we previously talked about (Integers, Floating numbers or Strings)
+
+[Image represents box analogy](https://www.hackerearth.com/practice/python/getting-started/python-variables/tutorial/)
+
+(In this picture, the box represents the variable named pi.Inside the variable pi we have stored the value 3.14.)
+
+
+a.Naming a variable:
+
+Naming a variable is important because it gives context to the store valuable that it represents. A variable can contain special characters, numbers and letters. The only thing to remember when naming a variable is that it cannot begin with a number or contain python keywords(refer to chart below). Once you name a variable you assign it to a single value using an equal sign. To the left of the equal sign is the variable name and to the right is the value stored in that variable. 
+
+
+Python keywords:
+
+[Python keywords chart](https://www.journaldev.com/13996/python-keywords-identifiers)
+
+
+Ex 2.1: 
+
+	cost = 6
+
+
+This is a correct way of naming a variable. Here we have assigned the variable name cost to the number 6 representing the cost of an item. The variable name cost is to the left of the equal sign and the value itself is to the right of the equal sign. 
+
+Ex 2.2:
+
+	6_price = 32
+
+
+Python would not accept this variable name because it doesn’t follow the naming rules. Remember what we had discussed earlier! When naming a variable we can’t begin with a number. Instead maybe we could write out the number 6 into six which would be acceptable. 
+
+Ex 2.3:
+
+	min = 2
+
+
+This is another variable name that Python will not accept. The variable name is min which is a Python keyword. If you need to please refer back to the Python keyword chart. 
+
+
+b.more variables:
+
+Remember that variables can be assigned to any kind of Python Data Type (integer, floating number, string). 
```

Comment:
add boolean to list if you're covering it

Clarify that there are other data types (lists, sets, dictionaries, etc.) that you did not cover but are valid assignments